conans/client/generators/__init__.py

```diff
@@ -137,6 +137,9 @@ def _new_generator(self, generator_name, output):
         elif generator_name == "BazelToolchain":
             from conan.tools.google import BazelToolchain
             return BazelToolchain
+        elif generator_name == "XCodeDeps":
```

Comment:
The name of the app is `Xcode`, not `XCode`, probably it should match

```suggestion
        elif generator_name == "XcodeDeps":
```